webapp/tests/karma/ts/services/contact-view-model-generator.service.spec.ts

```diff
@@ -182,11 +195,11 @@ describe('ContactViewModelGenerator service', () => {
     });
 
     it('if no contact person in children, persons still get displayed', () => {
-      stubLineageModelGenerator(null, doc);
+      stubLineageModelGenerator(doc);
       stubDbGet({ status: 404 }, childContactPerson);
-      stubSearch(null, []);
-      stubGetDataRecords(null, []);
-      stubDbQueryChildren(null, doc._id, [childPerson]);
+      stubSearch([]);
+      stubGetDataRecords([]);
+      stubDbQueryChildren(doc._id, [childPerson]);
       return service.getContact(doc._id)
         .then(waitForModelToLoad)
         .then(model => {
```

Comment:
For some reason I can't comment in the line. 
But in line 210 there is a unit test with wrong quote, can you please change from ` to ' (regular string quote) thanks!